### DISCUSSION TITLE
Netteté de l'image : 1 coup de scroll nécessaire

### DISCUSSION THREAD

**Discussion by LauwCost** [senshu/Sozi#631] [Q&A]
Bonjour à toutes et tous, 

Dernier petit post et après j'arrête de vous embêter. :)
Lorsque je navigue entre les [diapos produites](https://ecn.cemea.asso.fr/communs-numeriques/#diapo-accueil), je constate que le rendu est flou après chaque mouvement. 
Je suis obligé de donner un petit coup de scroll pour zoomer ou dézoomer et l'image devient alors nette.
Suis-je le seul à constater cela ? Quelle(s) préconisation(s) si vous convergez avec mon constat ?
Merci d'avance pour vos réponses, 
Librement, 
LauwCost

**Comment by aumouvantsillage** [comment]
Si vous utilisez Firefox, c'est un problème connu et qui a fait l'objet d'un rapport du bug : #464 
Pour moi, jusqu'à preuve du contraire, c'est un bug dans Firefox.
Mes efforts pour contourner le problème ont échoué

**Comment by LauwCost** [comment]
Bonjour Guillaume,

Merci beaucoup pour la célérité de la réponse et sa clarté.

Désolé de ne pas avoir trouvé ce signalement : c'est une pratique que je 
ne maîtrise pas totalement que de rechercher si le sujet a déjà été 
abordé et l'anglais me freine aussi.

Quoiqu'il en soit, je vais utiliser chromium (effectivement, je ne 
constate pas le soucis) pour les présentations en attendant jusqu'aux 
prochaines mise à jour de Firefox qui régleront peut-être le problème.

Merci encore !

C'est clos pour moi.

Utilisation de ma présentation aux Geek Faeries 
<https://www.geekfaeries.fr/> ce week-end :).

Librement

Le 01/06/2022 à 22:30, Guillaume Savaton a écrit :
